udata/core/reuse/models.py

```diff
@@ -23,6 +25,20 @@ class ReuseQuerySet(OwnedQuerySet):
     def visible(self):
         return self(private__ne=True, datasets__0__exists=True, deleted=None)
 
+    def visible_by_user(self, user: User) -> OwnedQuerySet:
+        """Return EVERYTHING visible to the user."""
+        public_qs: OwnedQuerySet = db.Q(private__ne=True, deleted=None, archived=None)
```

Comment:
Could we use `visible`?
```suggestion
        public_qs: OwnedQuerySet = db.Q(private__ne=True, deleted=None, archived=None)
```

Reply:
We could! It's a bit more work, but I believe it's less duplication this way, and I even moved the `visible_by_user` method to the `OwnedQuerySet` which is used by many models, so it'll be easier to reuse (pun not intended) in the future.